packages/android-client/FishjamClient/src/main/java/com/fishjamcloud/client/webrtc/PeerConnectionManager.kt

```diff
@@ -498,7 +517,7 @@ internal class PeerConnectionManager(
           return@launch
         }
       }
-      peerConnectionListener.onAddTrack(trackId!!, receiver!!.track()!!)
+      listeners.forEach { listener -> listener.onAddTrack(trackId!!, receiver!!.track()!!) }
```

Comment:
Could we refactor code to avoid `!!`?

Reply:
No, we probably can't. If something is null here that probably means something went very wrong and the backend gave us wrong data, so it's going to crash anyway later and we can't reasonably handle it. Let's crash earlier, so it's easier to debug.

Reply:
Hmm, I think we should not crash app in case of incorrect data. We can throw some error back to app, but crashing is something we should really avoid. (crashing app is worse UX than just showing error)